docs/tutorial/rust/src/main_game_logic_in_rust.rs

```diff
@@ -56,17 +56,17 @@ fn main() {
 }
 
 slint::slint! {
-    struct TileData := {
+    export struct TileData {
```

Comment:
Don't think this is needed. Note that for example `examples/memory/memory.slint` also doesn't export it.
```suggestion
   struct TileData {
```